### PR TITLE
heroku-logger: Fixed prefix type and return type from constructor

### PR DESCRIPTION
**Pull request by blackyale**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X ] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ianstormtaylor/heroku-logger#readme
- [X] Increase the version number in the header if appropriate.
- [] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/heroku-logger/index.d.ts [hunk 1]
```diff
@@ -12,28 +13,26 @@
...
-export function info(message: string, data?: object): void;
-export function warn(message: string, data?: object): void;
-export function error(message: string, data?: object): void;
-export function fatal(message: string, data?: object): void;
+ export function trace(message: string, data?: object): void;
+ export function debug(message: string, data?: object): void;
+ export function info(message: string, data?: object): void;
+ export function warn(message: string, data?: object): void;
+ export function error(message: string, data?: object): void;
+ export function fatal(message: string, data?: object): void;
```
Comment (kylevogt) [inline types/heroku-logger/index.d.ts]: Looks like white space was accidentally added here

**Review by kylevogt — APPROVED** [review]
Noted what appears to be accidental white spaces fixes make sense though.

**Review by kylevogt — APPROVED** [review]

File: types/heroku-logger/index.d.ts (hunk 1 above)
Comment (AlejandroMoran) [reply types/heroku-logger/index.d.ts]: Thanks, I think it has been fixed, could you check it as well?

Comment (kylevogt) [reply types/heroku-logger/index.d.ts]: Yup, looks much better, thanks for making the fixes! (original and white space)

Comment (AlejandroMoran) [reply types/heroku-logger/index.d.ts]: Thanks to you for creating this! :)

**Pull request merged**